src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.last {
+  margin-right: 50px;
```

Comment:
use pseudo-classes not to apply margin for the last element

Reply:
isn't fixed